docker/ci/dockerfiles/current/release.almalinux8.clients.x64.arm64.dockerfile

```diff
@@ -0,0 +1,141 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# This is a docker image specifically for standardize the ci/cd environment
+# for both developers and ci/cd pipeline for releasing Opensearch clients and other products
+# Please read the README.md file for all the information before using this dockerfile
+
+# Related dockerhub image: opensearchstaging/ci-runner:release-almalinux8-clients-v*
+
+
+FROM almalinux:8
+
+ARG MAVEN_DIR=/usr/local/apache-maven
+ARG CONTAINER_USER=ci-runner
+ARG CONTAINER_USER_HOME=/home/ci-runner
+
+# Ensure localedef running correct with root permission
+USER 0
+
+# Add normal dependencies
+RUN dnf clean all && \
+    dnf update -y && \
+    dnf install -y which curl git gnupg2 tar net-tools procps-ng python39 python39-devel python39-pip zip unzip jq
+
+# Add Python dependencies
+RUN dnf install -y @development zlib-devel bzip2 bzip2-devel readline-devel sqlite sqlite-devel openssl-devel xz xz-devel libffi-devel findutils
+
+# Add Yarn dependencies
+RUN dnf groupinstall -y "Development Tools" && dnf clean all && rm -rf /var/cache/dnf/*
+
+# Installing dotnet
+ARG DOT_NET_LIST="8.0"
+RUN for dotnet_version in $DOT_NET_LIST; do dnf install -y dotnet-sdk-$dotnet_version; done
+
+# Tools setup
+COPY --chown=0:0 config/yq-setup.sh config/gh-setup.sh /tmp/
+RUN dnf install -y go && /tmp/yq-setup.sh && /tmp/gh-setup.sh
+
+# Install JDK
+RUN curl -SL https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.15%2B10/OpenJDK11U-jdk_x64_linux_hotspot_11.0.15_10.tar.gz -o /opt/jdk11.tar.gz && \
+    mkdir -p /opt/java/openjdk-11 && \
+    tar -xzf /opt/jdk11.tar.gz --strip-components 1 -C /opt/java/openjdk-11/ && \
+    rm /opt/jdk11.tar.gz
```

Comment:
Let's remove this setup and use the config/jdk-setup.sh like any others.
Since this only has x64 arch not gonna support arm64 arch.

Reply:
Or you can have a branch to install corresponding jdk version based on arch.

Reply:
Sure changed to jdk-setup